xml/schema/Campaign/Survey.xml

```diff
@@ -214,6 +214,13 @@
     <default>NULL</default>
     <comment>Used to store option group id.</comment>
     <add>3.3</add>
+    <pseudoconstant>
+      <table>civicrm_option_group</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>title</labelColumn>
+      <nameColumn>name</nameColumn>
+      <prefetch>FALSE</prefetch>
```

Comment:
I checked the core code and it uses a pretty dumb hack which we might as well do here too (and then no need for disabling prefetch):

```suggestion
      <condition>name LIKE "civicrm_survey_%"</condition>
```

Reply:
This is the dumb hack I'm referring to: https://github.com/civicrm/civicrm-core/blob/0419bf7b41a6c5b5ac22a31bf0b0589dc649e1b3/CRM/Campaign/BAO/Survey.php#L694

Reply:
@colemanw ok - I've made that change